### DISCUSSION TITLE
Not seeing aria labels on day buttons

### DISCUSSION THREAD

**Discussion by iantbarker** [gpbl/react-day-picker#1545] [Support] [answered]
On the docs examples, the days buttons have an aria-label with the date on it, but on the newest versions, it doesn't have that label. Am I missing a setting to enable that or was this removed in some version? I tried different versions in a codesandbox and couldn't find one that includes the labels. It would really helpful to have those labels for jest testing purposes.

This is from a documentation instance:
<img width="517" alt="Screen Shot 2022-09-07 at 1 19 31 PM" src="https://user-images.githubusercontent.com/7671200/188950238-05aa721f-0f55-46ee-a7cf-b86f90522b44.png">

This is what is on 8.2:
<img width="523" alt="Screen Shot 2022-09-07 at 1 20 41 PM" src="https://user-images.githubusercontent.com/7671200/188950435-b6f14d33-43b1-44dd-beac-a4717e3de2f0.png">

**Comment by gpbl** [comment, marked as answer]
@iantbarker I can see the labels with the latest version: https://codesandbox.io/s/react-day-picker-8-vl4ry 

Could you replicate your issue by forking [this sandbox](https://codesandbox.io/s/react-day-picker-8-vl4ry)?

**Reply by iantbarker** [reply]
@gpbl I think I duplicated it by being on version 8.1.4 and react/react-dom at 16.9

https://codesandbox.io/s/react-daypicker-forked-0wdfne

**Reply by gpbl** [reply]
@iantbarker please upgrade to 8.2 – that attribute has been added from that version in https://github.com/gpbl/react-day-picker/pull/1537.

**Comment by driera** [comment]
Hi @gpbl, I see that from [last v8.5.0 update](https://github.com/gpbl/react-day-picker/releases/tag/v8.5.0), labelDay has been deprecated and aria-label removed. This causes each grid cell to be named by a number, [see sandbox](https://codesandbox.io/s/react-daypicker-forked-nmqsnz).

Prior to this change each grid cell stated the date in local format, which was nice in terms of a11y, [see sandbox](https://codesandbox.io/s/react-daypicker-forked-o57kh2)

Is there a reason why is the label being removed? Is there an alternative to print a localised date in the aria-label?


**EDIT**: There is a discussion that gives a clear reply about this: https://github.com/gpbl/react-day-picker/discussions/1716

**Reply by gpbl** [reply]
@driera sorry for the confusion about the `labelDay`. You can restore the previous behavior or use `labelDay` as you want by passing a custom `DayContent` component: https://github.com/gpbl/react-day-picker/issues/1774#issuecomment-1596892784

We will fix this better in the next major version. Thanks for your understanding.

**Reply by driera** [reply]
It's ok @gpbl. I had not thought about the custom `DayContent`, I'll give it a try. Thanks!
